system/ping/ping.c

```diff
@@ -224,6 +241,8 @@ int main(int argc, FAR char *argv[])
   info.delay     = ICMP_POLL_DELAY;
   info.timeout   = ICMP_POLL_DELAY;
   info.callback  = ping_result;
+  info.priv      = &priv;
+  priv.code      = ICMP_I_BEGIN;
```

Comment:
i feel it's cleaner to introduce a dedicated constant than using ICMP_I_BEGIN

Reply:
Fix